package.json

```diff
@@ -24,23 +24,27 @@
     "clean": "rimraf *.report.html *.report.dom.html *.report.json *.screenshots.html *.screenshots.json *.devtoolslog.json *.trace.json || true",
     "lint": "[ \"$CI\" = true ] && eslint --quiet -f codeframe . || eslint .",
     "smoke": "bash lighthouse-cli/test/smokehouse/run-all-tests.sh",
-    "coverage": "istanbul cover -x \"**/third_party/**\" _mocha -- $(find */test -name '*-test.js') --reporter progress --report lcovonly",
-    "coveralls": "yarn coverage && cat ./coverage/lcov.info | coveralls && ./node_modules/codecov/bin/codecov",
     "debug": "node --inspect-brk ./lighthouse-cli/index.js",
     "start": "node ./lighthouse-cli/index.js",
     "test": "yarn lint --quiet && yarn unit && yarn type-check && yarn closure",
     "test-extension": "cd lighthouse-extension && yarn test",
-    "unit-core": "bash lighthouse-core/scripts/run-mocha.sh --core",
-    "unit-cli": "bash lighthouse-core/scripts/run-mocha.sh --cli",
-    "unit-viewer": "bash lighthouse-core/scripts/run-mocha.sh --viewer",
-    "unit": "bash lighthouse-core/scripts/run-mocha.sh --default",
+
+    "unit-core": "mocha --reporter dot \"lighthouse-core/test/**/*-test.js\"",
+    "unit-cli": "mocha --reporter dot \"lighthouse-cli/test/**/*-test.js\"",
+    "unit-viewer": "mocha --reporter dot \"lighthouse-viewer/test/**/*-test.js\"",
+    "unit": "yarn unit-core && yarn unit-cli && yarn unit-viewer",
     "core-unit": "yarn unit-core",
     "cli-unit": "yarn unit-cli",
     "viewer-unit": "yarn unit-viewer",
+    "watch": "yarn unit-core --watch",
+
+    "unit:silentcoverage": "nyc --silent yarn unit",
```

Comment:
why silent coverage? just got annoyed with it in output?

Reply:
yeah here's an example: https://travis-ci.org/GoogleChrome/lighthouse/builds/361959444?utm_source=github_status&utm_medium=notification